XRTK.Oculus/Packages/manifest.json

```diff
@@ -1,9 +1,17 @@
 {
+  "scopedRegistries": [
+    {
+      "name": "XRTK",
+      "url": "http://upm.xrtk.io:4873/",
+      "scopes": [
+        "com.xrtk"
+      ]
+    }
+  ],
   "dependencies": {
     "com.unity.package-manager-ui": "2.1.2",
     "com.unity.textmeshpro": "2.0.1",
-    "com.xrtk.core": "https://github.com/XRTK/XRTK-Core.git#0.1.16",
-    "com.xrtk.upm-git-extension": "https://github.com/XRTK/UpmGitExtension.git#0.9.6",
+    "com.xrtk.core": "0.1.19",
```

Comment:
let's add the sdk as well

Reply:
I've added it as requested, but none of the other extensions are including SDK